erc20-price/proto/v1/chainlink.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package messari.chainlink.v1;
+
+message Aggregator {
+  string address = 1;
+  string description = 2;
+  string base = 3;
+  string base_address = 4;
+  string quote = 5;
```

Comment:
I think we should still have quote address for the general case. Consider our use of chainlink price feed for the price module as a special use case of the chainlink aggregator, and there could be other use cases that we may not know about that could require the quote address